### DISCUSSION TITLE
Skimmer dismounting

### DISCUSSION THREAD

**Discussion by MohAl-s** [Friendly0Fire/GW2Radial#365] [General]
Hiya, so on the previous version whenever you click your radial button you mount and click it again you dismount both on land and in water. However now when I tried to mount up then dismount on land it worked the same way but in water after i get on my skimmer I am unable to dismount again unless I use the gw2 in game button. Has the settings been moved around a bit or is it just not an option anymore.

**Comment by Friendly0Fire** [comment]
Make sure you have the generic mount/dismount button set in your keybinds in the addon, it makes dismounting a lot more reliable.

**Comment by MohAl-s** [comment]
Oh I see what you mean so set the mount/dismount button to the in game one and everything should work?
Edit: Works perfect TYTY
